be/src/storage/vectorized/column_predicate_rewriter.h

```diff
@@ -77,4 +79,18 @@ class ConjunctivePredicatesRewriter {
     std::vector<uint8_t>* _disable_dict_rewrite;
 };
 
+class ZonemapPredicatesRewriter {
+public:
+    using PredicateList = std::vector<const ColumnPredicate*>;
+
+    static Status rewrite(ObjectPool* pool, const std::unordered_map<ColumnId, PredicateList>& src,
```

Comment:
what's the difference of two `rewrite` method. would better give a more concrete method name